v2/fleet/sumologic.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Sumologic Collector
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=core
+Restart=always
+ExecStartPre=/usr/bin/docker pull behance/docker-sumologic
+ExecStartPre=-/usr/bin/docker stop sumologic
+ExecStartPre=-/usr/bin/docker rm -f sumologic
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill sumologic
+ExecStartPre=-/usr/bin/docker rm sumologic
```

Comment:
Sumologic and sumologic-control are 2 services to allow us to log 2 things to different source categories. They's don't run on the same tiers.